### DISCUSSION TITLE
Event registration form: Which fields & hooks do you use?

### DISCUSSION THREAD

**Discussion by oliverklee** [oliverklee/ext-seminars#1376] [Ideas]
As part of the [compatibility with TYPO3 V11](https://github.com/oliverklee/ext-seminars/wiki/11LTS-Crowdfunding-Campaign), I will completely rewrite the frontend event registration form. For this, I plan to only implement the inputs and displays and hooks that are actually still used/needed. For this, I need your input. :-)

If you are using the event registration form (with which people can register for events), please comment:

- Which of the existing inputs in the event registration form do you use (or plan to use)?
- Which of the existing displays in the event registration form do you use (or plan to use)?
- Which of the existing hooks of the event registration do you use? For which purpose?

**Comment by lisardo2** [comment]
Which of the existing inputs in the event registration form do you use (or plan to use)?
- company
- name
- address
- zip
- city 
- email
- interests
- seats
- expectations
- notes
- background knowledge
- known from
- submit

Which of the existing displays in the event registration form do you use (or plan to use)?
- we overwrote the template completely, so ein dont know exactly what you mean

Which of the existing hooks of the event registration do you use? For which purpose?
- none. but i presume they could become valuable - please dont drop them.

**Comment by t3premium** [comment]
We use the following fields in a special use case with a global frontend user:

- registered_themselves
- attendees_names
- seats

**Comment by oliverklee** [comment]
Note from a customer email: When the checkbox "registered themselves" is not displayed, the value of this (not displayed) field needs to be configurable.

**Comment by peerlight** [comment]
## Which of the existing inputs in the event registration form do you use (or plan to use)?

- price,method_of_payment, company, name, address, zip, city, country, telephone, email,registered_themselves, notes, total_price,lodgings, feuser_data, registration_data, terms, terms_2

## Which of the existing displays in the event registration form do you use (or plan to use)?

- the summary of the entered data
- options of event
- terms 
- the UID and title of the event

## Which of the existing hooks of the event registration do you use? For which purpose?

- none

**Comment by oliverklee** [comment]
@peerlight Thanks!

With "displays", I'm referring to:
- the summary of the entered data
- the FE user data coming from the logged-in user (not the data copied to the billing address fields by default)
- the step counter
- the UID and title of the event

**Comment by jpmschuler** [comment]
As we just have the most basic use of fe_login, select event (no use of actual dates, prices or anything), add notes (no fields except a free field) and join waiting queue, thus:
We only details from fe_users (name, mail, custom fields) to help users understand that they were logged in correctly and what data is transferred and have a field for notes.

**Comment by oliverklee** [comment]
From a customer:
- event overview (title, date)
- comments
- summary
- terms and terms_2
- separate billing address
- hooks for the registration emails
